packages/client/src/lib.js

```diff
@@ -130,9 +175,7 @@ class Web3Storage {
   // Just a sugar so you don't have to pass around endpoint and token around.
 
   /**
-   * Stores files encoded as a single [Content Addressed Archive
-   * (CAR)](https://github.com/ipld/specs/blob/master/block-layer/content-addressable-archives.md).
-   *
+   * Puts files.
```

Comment:
The previous comment tells the user a bit more about what is gong on
```suggestion
   * Uploads files to web3.storage. Files are hashed in the client and uploaded as a single 
   * [Content Addressed Archive(CAR)](https://github.com/ipld/specs/blob/master/block-layer/content-addressable-archives.md).
```